stl/inc/format

```diff
@@ -1878,7 +1892,16 @@ private:
             _Arg_type = _Basic_format_arg_type::_Custom_type;
         }
 
-        _Store_impl<_Erased_type>(_Arg_index, _Arg_type, static_cast<_Erased_type>(_Val));
+#if !_HAS_CXX23
+        // Workaround towards N4928 [format.arg]/9 and /10 in C++20
+        if constexpr (is_same_v<_Erased_type, basic_string_view<_CharType>>) {
+            typename _Traits::template _Converted_string_type<_Ty> _Str = _Val;
+            _Store_impl<_Erased_type>(_Arg_index, _Arg_type, _Erased_type{_Str.data(), _Str.size()});

```

Comment:
Can't we directly use `_Val`, since we know that `_Val` is either a `basic_string_view` or a `basic_string`?

```suggestion
            _Store_impl<_Erased_type>(_Arg_index, _Arg_type, _Erased_type{_Val.data(), _Val.size()});
```

Reply:
Perhaps it's OK to directly use `_Val`. I was afaid of derived types from `basic_string(_view)` with `formatter` specializations for them, but I think these types should be caught by `handle`/`_Custom_type` now.